backend/src/main/services/EventService.ts

```diff
@@ -24,7 +30,29 @@ export class EventService {
       throw new HttpException(StatusCodes.BAD_REQUEST, 'Start and End date not valid');
     }
 
-    return this.eventRepository.create(eventCreationDTO);
+    const newEvent = await this.eventRepository.create(eventCreationDTO);
+
+    eventCreationDTO.invitee.forEach(async (invitee) => {
+      await this.emailService.sendEmail(
+        invitee,
+        'Meeting Invitation',
+        `You are invited to join the following meeting: <br><br>
+        Created By: ${eventCreationDTO.createdBy} <br>
+        Title: ${eventCreationDTO.title} <br>
+        ${eventCreationDTO.description ? `Description: ${eventCreationDTO.description} <br>` : ``}
+        ${eventCreationDTO.location ? `Location: ${eventCreationDTO.location} <br>` : ``}
+        Start date: ${new Date(eventCreationDTO.start).toLocaleString()} <br>
+        End date: ${new Date(eventCreationDTO.end).toLocaleString()} <br><br>
+        <p>Click <a href="${process.env.FRONTEND_DOMAIN}event/invitation/status?id=${
```

Comment:
The FRONTEND_DOMAIN value in the .env file should end with a '/' right?

Reply:
Yes, I'll send the new .env since there are a few new environment variables.